frontend/README.md

```diff
@@ -1,34 +1,53 @@
 # The Urlist - Frontend
+
 [![Build status](https://burkeknowswords.visualstudio.com/The%20Urlist/_apis/build/status/Frontend%20Build)](https://burkeknowswords.visualstudio.com/The%20Urlist/_build/latest?definitionId=7)
 
-The frontend for this project was build with the following libraries and frameworks:
+The frontend has the following views:
+
+- **Homepage view**: this is the first page you would see when visiting the frontend. You can start the process of creating a link-bundles from here.
+
+  ![Homepage picture](docs/Images/Homepage.png)
+
+- **Edit view**: this is the page where you would create your lists.
+  
+  ![Edit picture](docs/Images/Edit_page.png)
+
+- **User view**: this page would list all list-bundles created for a signed-in user.
+  
+  ![Manage lists picture](docs/Images/Manage_lists_page.png)
+
+- **List view**: this page is used to view the list-bundle for a given vanity URL.
+
+  ![View list picture](docs/Images/View_page.png)
+
+The frontend for this project was built with the following libraries and frameworks:
 
-* [TypeScript](https://www.typescriptlang.org/)
-* [Vue.js](https://github.com/vuejs/vue) / [Vue CLI](https://github.com/vuejs/vue-cli)
-* [Vuelidate](https://github.com/vuelidate/vuelidate)
-* [Axios](https://github.com/axios/axios)
+- [TypeScript](https://www.typescriptlang.org/)
+- [Vue.js](https://github.com/vuejs/vue) / [Vue CLI](https://github.com/vuejs/vue-cli)
+- [Vuelidate](https://github.com/vuelidate/vuelidate)
+- [Axios](https://github.com/axios/axios)
 
 Other useful tools
 
-* [Visual Studio Code](https://code.visualstudio.com/?WT.mc_id=theurlist-github-buhollan)
-* [Vetur](https://marketplace.visualstudio.com/items?itemName=octref.vetur&WT.mc_id=theurlist-github-buhollan)
-* [VS Code Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome&WT.mc_id=theurlist-github-buhollan)
-* [Vue VS Code Extension Pack](https://marketplace.visualstudio.com/items?itemName=sdras.vue-vscode-extensionpack&WT.mc_id=theurlist-github-buhollan)
-* [Vue browser devtools](https://github.com/vuejs/vue-devtools)
+- [Visual Studio Code](https://code.visualstudio.com/?WT.mc_id=theurlist-github-buhollan)
+- [Vetur](https://marketplace.visualstudio.com/items?itemName=octref.vetur&WT.mc_id=theurlist-github-buhollan)
+- [VS Code Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome&WT.mc_id=theurlist-github-buhollan)
+- [Vue VS Code Extension Pack](https://marketplace.visualstudio.com/items?itemName=sdras.vue-vscode-extensionpack&WT.mc_id=theurlist-github-buhollan)
+- [Vue browser devtools](https://github.com/vuejs/vue-devtools)
 
 ## <a name="feconfigs" ></a> Frontend configurations
 
 There are two configurations needed for the frontend to run, those are passed as [VUE environment variables](https://cli.vuejs.org/guide/mode-and-env.html)

```

Comment:
There are now more than 2 with the introduction of the OIDC var below.

Reply:
Good catch, I will check the auth code and update accordingly.